test/automation/src/sql/notebook.ts

```diff
@@ -132,6 +136,17 @@ export class Notebook {
 		await this.code.waitForElement(expectedResultSelector);
 	}
 
+	async isCodeCellInEditMode(): Promise<boolean> {
+		// verify that cell is in editmode
+		const element = await this.code.waitForElement(`.notebook-cell.active .monaco-editor`);
+		return element.className.includes('focused');
```

Comment:
Is there not some other way to tell this? I guess checking if the editor is focused is enough, but a comment here explaining why being focused implies "edit mode" would be helpful